src/components/PokemonSearchBarResults/PokemonResults.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { describe, it, expect } from 'vitest';
+import PokemonResults from './PokemonResults';
+import { mockPokemon, mockPokemon2 } from '../../test-utils/mockData';
+
+describe('PokemonResults', () => {
+  it('displays loader when loading', () => {
```

Comment:
you already checked it in test about, but you can leave it here for cross check review